extend-vnet-to-multi-vnet/azuredeploy.json

```diff
@@ -0,0 +1,286 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "virtualNetworkName1": {
```

Comment:
Since this is an existing VNET, it'd be better if this parameter has an 'existing' prefix to set the right expectations.